src/Event Controllers/main.js

```diff
@@ -48,33 +36,41 @@ const gesture_click = new Gtk.GestureClick({ button: 0 });
 window.add_controller(gesture_click);
 
 gesture_click.connect("pressed", (_self, _n_press, _x, _y) => {
-  switch (gesture_click.get_current_button()) {
+  let css_class = "suggested-action";
+  if (_self.get_current_event_state() & Gdk.ModifierType.CONTROL_MASK) {
```

Comment:
The `_self` variable should have the leading underscore removed, if it's being used now.